tests/functional/api/test_groups.py

```diff
@@ -81,9 +81,10 @@ def test_it_adds_member_to_group(self, app, user, group, auth_client_header):
 
         assert user in group.members
 
-    def test_it_ignores_forwarded_user_header(self, app, user, factories, group, auth_client_header):
+    def test_it_ignores_forwarded_user_header(self, app, user, factories, group, auth_client_header, db_session):
         headers = auth_client_header
         user2 = factories.User()
+        db_session.commit()
```

Comment:
This test was actually broken in previous PR—it didn't test what I thought it did. It does now.